### PR TITLE
Inline Docs: nav-main partial

### PR DESCRIPTION
**Pull request by dryanmedia**
Inline docs for the Main Navigation

### CODE REVIEW

File: partials/nav-main.php [hunk 1]
```diff
@@ -1,55 +1,79 @@
...
+ * This is the primary navigation for "verticals" on a homepage. Other pages use 
+ * sticky navigation if enabled.
+ *
+ * @link URL
+ *
+ * @package Largo
+ *
+ * If is home page OR sticky nav is disabled show this Main Navigation.
+ */
 if ( is_front_page() || is_home() || !of_get_option( 'show_sticky_nav' ) ): ?>
```
Comment (aschweigert) [inline partials/nav-main.php]: idk about this, don't we have another pull somewhere to add a convenience function or something more sane here?

Comment (dryanmedia) [inline partials/nav-main.php]: @aschweigert this convenience function is now in #916. I'm going to close this pull and reopen submitting to develop. Once #916 is in the replacement pull for this and additionally #912 can both adopt the convenience function.

**Comment by aschweigert** [comment]
sounds good, please assign the new PR to ryan to review

**Comment by rnagle** [comment]
Did a little work to clean this up before realizing it's already been merged into develop. Looks like @dryanmedia filed another PR against `develop`. Closing...

**Pull request closed without merging**